locale/en/LC_MESSAGES/django.po

```diff
@@ -46,23 +171,23 @@ msgstr "Any authenticated student"
 
 #: authorization/permissions.py
 msgid "ACCESS_ENROLLED_STUDENT"
-msgstr "Enrolled student of the course"
+msgstr "An enrolled student of the course"
 
 #: authorization/permissions.py
 msgid "ACCESS_COURSE_ASSISTANT"
-msgstr "Assistant of the course"
+msgstr "An assistant of the course"
 
 #: authorization/permissions.py
 msgid "ACCESS_GRADING"
-msgstr "Grading. Assistant if course has that option or teacher"
+msgstr "Grading. An assistant if course has that option or a teacher"
```

Comment:
I'm not quite sure what this string means. I would use commas, e.g. "An assistant, if course has that option, or a teacher"

Reply:
I'm not sure in which contexts it's used (basically it's a description of the access level, so if the course allows TA's to grade, if the access level is "GRADING", both teachers and TA's can access the page or whatever, if TA's aren't allowed grading, it's just teachers). Good point about the commas. Do you think knowing the context now that commas or parentheses would be better, for example
"An assistant, if the course has allowed that option, or a teacher" or "An assistant, if the course has allowed assistant grading, or a teacher"
"An assistant (if the course has allowed that option) or a teacher" or "An assistant (if the course has allowed assistant grading) or a teacher"

Reply:
Suggestion:
_“Member of course staff with grading rights: a teacher or (if allowed by course settings) an assistant.”_ 

BTW, the existing text seems to suggest that TA grading is set at a course level when in fact it can be adjusted at a much finer grain, even for individual assignments.